src/terraboot/infra.clj

```diff
@@ -1,10 +1,16 @@
 (ns terraboot.infra
   (:require [terraboot.core :refer :all]
             [terraboot.vpc :refer [vpc-vpn-infra vpc-name]]
-            [terraboot.cluster :refer [cluster-infra]]))
+            [terraboot.cluster :refer [cluster-infra]]
+            [clojure.edn :as edn]))
 
 (def infra-path "infra/")
 
+(defn get-config
+  "Gets info from a config file."
+  [url]
+  (edn/read-string (slurp url)))
+
 (defn generate-json [target]
   (let [account-number "165664414043"
```

Comment:
That's the thing, I would advocate for the infra.clj part to be in a private repo.  terraboot lib public, all specific data private